unityfiles/Assets/Scripts/Core/LevelManager.cs

```diff
@@ -65,6 +69,11 @@ public class LevelManager : Singleton<LevelManager> {
     private void UnsubscribeFromEvents() {
         Player.PlayerKilledEvent -= CallbackPlayerKilledEvent;
         Enemy.EnemyKilledEvent -= CallbackEnemyKilledEvent;
+        PauseMenuController.PauseMenuChangeEvent -= CallbackChangeLevelState;
+    }
+
+    private void CallbackChangeLevelState(object _, PauseMenuChangeEventArgs args) {
+        ChangeLevelState(args.NewLevelState);
     }
```

Comment:
denne "burde" ikke finnes på en måte, selv om dette er hva nesten alle eventene inn jør er det gode (debug) grunder til og gå indirekte gjennom en fungsjon som de to over